src/wp-includes/rest-api/endpoints/class-wp-rest-post-types-controller.php

```diff
@@ -394,4 +385,22 @@ public function get_collection_params() {
 		);
 	}
 
+	/**
+	 * Prepares links for the request.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param WP_Post_Type $post_type The post type.
+	 * @return array[]
+	 */
+	protected function prepare_links( WP_Post_Type $post_type ) {
```

Comment:
```suggestion
	protected function prepare_links( $post_type ) {
```

Let's avoid this for now.